### ISSUE TITLE
[A11y] Desktop/Mobile Search Results - Grid: Checkbox

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1828]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Search Results Grid: https://staging.oregondigital.org/catalog?locale=en&q%5B%5D=test&search_field=all_fields&utf8=%E2%9C%93&view=gallery

I am not logged in. When I tab through the images there are checkboxes that appear  on the first two thumbnails with the black rollover state and there is also a checkbox that appears at all times. See image below:

<img width="1012" alt="Screen Shot 2021-08-09 at 7 40 45 AM" src="https://user-images.githubusercontent.com/8353860/128724944-4242fca9-740b-4e3b-a1cf-9ca66878d8aa.png">

However, when I tab through the images with the black rollover state and try and check the checkbox. Nothing happens and same with the check box on the right that has the checkbox visible at all times.

As a user who is not logged in. Am I suppose to see those checkboxes and interact with them? This happens in both views grid and list.

### Suggested Fix

Have the checkboxes be clickable to the user by tabbing and pressing enter.

**Comment by CGillen** [comment]
Fixed by #2041
